README.md

```diff
@@ -1,73 +1,60 @@
-<!-- TABLE OF CONTENTS -->
-## Table of Contents
 
-* [About the Project](#about-the-project)
-* [Built With](#built-with)
-* [Getting Started](Getting-Started)
-* [Usage](#usage)
-* [Author](#author)
-* [Acknowledgements](#acknowledgements)
+# Project Name
+Robotics Team, for Microverse Copstone project-1
 
-<!-- ABOUT THE PROJECT -->
-## About The Project
+> A description of the project here
+<img width="1438" alt="Screen Shot 2022-06-24 at 2 23 25 AM" src="https://user-images.githubusercontent.com/53530780/175408052-2b92a7e2-a1fd-46fb-831d-69dfa8c51b6b.png">
 
-Type about project
 
-### Built With
-This project was built using these technologies.
-* HTML
-* CSS3
-* VsCode
+## Live Demo
+[Live Demo Link](https://shahira-sadat.github.io/Capstone-1/)
 
+## Built With
+- html
+- css
+- node
+- git
+- bootstrap
+- VS Code
+- JS
 
-## Getting Started
+## Link to video
+[Loom video Link](https://shahira-sadat.github.io/Capstone-1/)
```

Comment:
- [ ] Kindly add a loom video presenting features of your project. Remember to highlight about a piece of the code that you find interesting or you are proud of making. This is project requirement. The current link does not have a video.

Reply:
Added link to loom video
https://www.loom.com/share/53267764b71b445fa9d28f212ceff507